README.md

```diff
@@ -0,0 +1,9 @@
+## Build
+
+```
+$ ./autogen.sh
```

Comment:
> `make dist` ... is how the release tarball should be created

Agreed - that was my plan. But thanks to having a conda package for this library I've not got around to doing it yet.